test/test_nestedtensor.py

```diff
@@ -1544,6 +1610,39 @@ def rand_mask(size):
             torch.ops.aten._scaled_dot_product_attention(
                 query, key, value, dropout_p=dropout_p, need_attn_weights=need_attn_weights, is_causal=True)
 
+    @dtypes(torch.float, torch.float16, torch.double)
+    def test_empty_like(self, device, dtype):
+        ntensors = 4
+        nt = self.random_nt(device, dtype, ntensors, (4, 4))
```

Comment:
I would use get contiguous and non_contiguous and make sure that non_contiguous errors